usage/index.adoc

```diff
@@ -1,14 +1,31 @@
-= Usage
+= Usage and Error reporting
 :page-layout: project
 :page-status: green
 
-Since medio August 2010 JBoss Tools 3.2 builds now provides a "Usage"
-plugin which when installed and enabled will help us gather anonymous
-statistics about how the JBoss Tools plugins are being used. All the
-data is gathered anonymously and can at any time be disabled if you
-wish.
+Our Eclipse plugins are constantly evolving and in that we are using
+usage data and error reports to know how the plugins are used and what
+areas that could use improvements.
 
-== What will we use the data for
+To do that there are two separate functionalities:
+
+<<Usage>>::
+  Since medio August 2010 JBoss Tools 3.2 provides a "Usage"
+  plugin which when installed and enabled will help us gather anonymous
+  statistics about how the JBoss Tools plugins are being used. All the
+  data is gathered anonymously and can at any time be disabled if you
+  wish.
+
+<<Error reporting>>::
```

Comment:
When rendered on github this title appears to have square braces around it. Might just be a formatting glitch on github.